### ISSUE TITLE
consult-line+embark-collect-snapshot has a tofu problem

### ISSUE THREAD

**Issue by minad** [oantolin/embark#331]
1. Search using consult-line
2. embark-collect-snapshot
3. Act on some match (jump to the original window)
4. Move some lines around
5. Acting again on other matches won't work anymore since the tofus are invalidated because of the line number changes

I thought we had handled the tofus? Or did some of the recent refactorings break this?

**Comment by minad** [comment]
Btw, I wonder if there is not a simpler solution for the problem. We should probably replace the consult-line default action with a default override which directly jumps to the correct line.

**Comment by minad** [comment]
I think the problem is this:

~~~ elisp
(defun embark-consult--accept-tofu (&rest _)
  "Accept input if it already has the unicode suffix.
This is intended to be used in `embark-setup-action-hooks' for the
`consult-line' and `consult-outline' actions."
  (let* ((input (minibuffer-contents))
         (len (length input)))
    (when (and (> len 0)
               (<= consult--tofu-char
                   (aref input (- len 1))
                   (+ consult--tofu-char consult--tofu-range -1)))
      (add-hook 'post-command-hook #'exit-minibuffer nil t))))
~~~

As I see it, the tofu acceptor should go away. But we want to keep the tofu stripper.

**Comment by minad** [comment]
Fixed by #332.
